pkg/cassandra/session.go

```
@@ -190,6 +190,12 @@ func isSelect(query string) bool {
 	return true
 }
 
+// stripKeyspace removes the keyspace qualifier from a fully qualified table
+// name, so that "metrics.temperature" becomes "temperature".
+func stripKeyspace(qualified, keyspace string) string {
```

golangci-lint (2.12.2)

[error] func stripKeyspace is unused

(unused)